core/common/src/main/java/alluxio/underfs/MultiRangeObjectInputStream.java

```diff
@@ -34,6 +33,9 @@
   /** Position the current stream was open till (exclusive). */
   protected long mEndPos;
 
+  /** The block size to read with. */
+  protected long mBlockSize = -1L;
```

Comment:
Why is this necessary?

Reply:
Previously we used the block size to perform reads. To keep the same behavior we need the `mBlockSize` parameter.

Reply:
https://github.com/Alluxio/alluxio/blob/master/core/common/src/main/java/alluxio/underfs/MultiRangeObjectInputStream.java#L115-L141

Reply:
Got it, thanks. Could we omit the parameter in `createStream` and use the member variable `mBlockSize` in the `end` calculation?

Reply:
sgtm